odoo/tests/common.py

```diff
@@ -2060,7 +2060,7 @@ def start_tour(self, url_path, tour_name, step_delay=None, **kwargs):
             'keepWatchBrowser': kwargs.get('watch', False),
             'debug': kwargs.get('debug', False),
             'startUrl': url_path,
-            'delayToCheckUndeterminisms': kwargs.pop('delay_to_check_undeterminisms', 0),
+            'delayToCheckUndeterminisms': kwargs.pop('delay_to_check_undeterminisms', os.getenv("ODOO_TOUR_DELAY_TO_CHECK_UNDETERMINISMS") or 0),
```

Comment:
the second argument of getenv is the default if the key doesn't exist
also, given the example below
you should pobably cast into a int


```
$ export LOL=4
$ ipython3
In [1]: import os

In [2]: os.getenv("LOL")
Out[2]: '4'
```
so:
```py

int(os.getenv("VARNAME", "0"))
```